src/auditlog/diff.py

```diff
@@ -135,7 +135,7 @@ def model_instance_diff(old, new):
         new_value = get_field_value(new, field)
 
         if old_value != new_value:
-            diff[field.name] = (smart_text(old_value), smart_text(new_value))
+            diff[model_fields['mapping_fields'].get(field.name, field.name)] = (smart_text(old_value), smart_text(new_value))
```

Comment:
I would not use the mapping here. I think keeping the database representation as close to internal representation as possible is a good idea, especially because later on we are using database values to retrieve model fields. Changing the mapping would completely destroy that capability for the changed fields. The actual field names are less likely to change.

The mapping could be applied when the dictionary for display is built up (see my comment there for details).

Reply:
Ah yes I agree, I reverted this change and now only change the name in the `changes_display_dict` property